documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,28 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (Both optional):
```

Comment:
I looked on the preview and it seems to be a bit mess.
Looks like there are 2 steps under VMware source title, which one is RHV only, later on there is a title of RHV source and there is the LUKs which is VMware source.

Reply:
Fixed; thanks.